### PR TITLE
Fix inconsistent use of tabs and spaces

### PR DESCRIPTION
**Pull request by dlangille**
I saw this while building:

*** Sorry: TabError: inconsistent use of tabs and spaces in indentation (customfunctions.py, line 17)

### CODE REVIEW

File: examples/warntoggle/mqttwarn/customfunctions.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def togglestate(topic, payload, section, srv):
         if topic in toggles:
             # file found, topic found
             topicblock = toggles[topic]
-	        srv.logging.debug('togglestate() was called from the ' + section + ' section and found ' + topic + ' in ' + filename)
+	    srv.logging.debug('togglestate() was called from the ' + section + ' section and found ' + topic + ' in ' + filename)
```
Comment (jpmens) [inline examples/warntoggle/mqttwarn/customfunctions.py:17]: it looks to me as though there are a few spaces missing there ... the `srv.` ought to be under `topicblock`

**Comment by amotl** [comment]
Dear Dan,

thanks for caring about these details. Will you be able to update the PR like outlined by @jpmens?

Thanks already and with kind regards,
Andreas.

**Comment by amotl** [comment]
This has been improved within 28e94649 - thanks again, @dlangille!

**Comment by dlangille** [comment]
Sorry, I've been preoccupied. :/

**Pull request closed without merging**